.github/workflows/reusable-agent-build-container-images.yml

```diff
@@ -474,6 +477,10 @@ jobs:
         run: |
           export RETRY_ATTEMPTS="14"
           export SLEEP_DELAY="10"
+          
+          # Wait for at least one hello-printer job to be scheduled and completed. 
+          RETRIES=1; 
+          until echo -n "Waiting for first job to be completed - retry $RETRIES: " && kubectl wait --for=condition=complete job -l app=hello-printer || [ $RETRIES -gt 60 ]; do sleep 1; RETRIES=$(($RETRIES+1)); done
```

Comment:
So it can take up to 1 minute for one pod to become scheduled and ready?

Reply:
We are waiting for a cronjob to start a first job.
The smallest scheduled cronjob time is "every minute".